ui/lib/apps/InstanceProfiling/pages/Detail.tsx

```diff
@@ -28,6 +46,20 @@ function mapData(data) {
       }
       task.progress = progress
     }
+
+    // set profiling output options for previous generated SVG files and protobuf files.
+    if (task.profile_output_type === ProfilingOutputTypeProtobuf) {
+      task.view_default_output_type_val = t(viewDefaultOutputTypeVal)
+    } else {
+      switch (task.target.kind) {
+        case 'tidb':
+        case 'pd':
+          task.view_default_output_type_val = t(ViewOutputTypeOptions.Graph)
+          break
+        default:
```

Comment:
This is not actually the default. It is not *semantically reasonable* to make "TiKV" and "TiFlash" to be default while TiDB is not a default. You will find `default` can work. But they are not directly related.